frontend/src/component/admin/network/NetworkTrafficUsage/RequestSummary.tsx

```diff
@@ -100,7 +102,33 @@ export const RequestSummary: FC<Props> = ({
                     <Row>
                         <dt>Included in your plan monthly</dt>
                         <dd>
-                            {includedTraffic.toLocaleString('en-US')} requests
+                            {(includedTraffic * 1_000_000).toLocaleString(
```

Comment:
I think maybe there's a constant for the traffic bundle size 1_000_000 somewhere